File: components_page/components/layout/no_gutters.py
import dash_bootstrap_components as dbc
from dash import html

row = dbc.Row(
    [
        dbc.Col(html.Div("One of three columns")),
        dbc.Col(html.Div("One of three columns")),
        dbc.Col(html.Div("One of three columns")),
    ],
    className="g-0",
)

File: components_page/components/layout/no_gutters.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

row_no_gutters <- dbcRow(
  list(
    dbcCol(htmlDiv("One of three columns"), className = "border bg-light"),
    dbcCol(htmlDiv("One of three columns"), className = "border bg-light"),
    dbcCol(htmlDiv("One of three columns"), className = "border bg-light")
  ),
  no_gutters = TRUE,
)
